web/src/components/TaskBody.tsx

```diff
@@ -1,31 +1,29 @@
 import { TaskElement } from "@/components/TaskElement";
 import { UnlockedBy } from "@/components/tasks/UnlockedBy";
-import { useRoadmap } from "@/lib/data-hooks/useRoadmap";
-import { useUserData } from "@/lib/data-hooks/useUserData";
 import { getNaicsDisplayMd } from "@/lib/domain-logic/getNaicsDisplayMd";
-import { Task } from "@/lib/types/types";
+import { Roadmap, Task } from "@/lib/types/types";
 import { templateEval } from "@/lib/utils/helpers";
 import { getModifiedTaskContent } from "@/lib/utils/roadmap-helpers";
+import { Business } from "@businessnjgovnavigator/shared/";
 import { ReactElement } from "react";
 
 interface Props {
   task: Task;
+  business?: Business;
+  roadmap?: Roadmap;
```

Comment:
Is there a case where these would be undefined? If the user can see a task screen wouldn't these be required?

Reply:
I'm assuming that this is partially here to support the CMS implementation.

Reply:
I do really like this change. Passing these in makes so much more sense!

Reply:
It's b/c we don't have a loading state when business or user data is undefined. I'm going to ask content if a loading state can be added to make it a required prop